perf_test/sparse/KokkosSparse_sptrsv_cholmod.cpp

```diff
@@ -595,7 +595,7 @@ int main(int argc, char **argv)
   return 0;
 }
 #else // defined(KOKKOSKERNELS_ENABLE_TPL_CHOLMOD)
-int main(int argc, char **argv)
+int main(int /*argc*/, char **/*argv*/)
```

Comment:
Actually in the case of a few of these `int main(int, char**)` I will just go with `int main()` since it will not impact anyone to remove the parameters altogether in that case.